src/components/ContentWidgets/Separator/SeparatorEditor.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { Separator } from "./Separator";
+
+/**
+ * SeparatorEditor edits the width and color of a separator widget
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 10, 2019
+ */
+export const SeparatorEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  const defaultColor = { r: 66, g: 13, b: 171 };
+  const width = editedContent.separator_width || 33;
+  const color = editedContent.separator_color || defaultColor;
+
+  /**
+   * Updates seperator_color
+   * @param e event
+   * @param c color: {red, green, blue}
+   */
+  let changeColor = (e: any, c: string) => {
+    let newColor = { ...color };
+    if (c === "red") {
+      newColor.r = e.target.value;
+    } else if (c === "green") {
+      newColor.g = e.target.value;
+    } else if (c === "blue") {
+      newColor.b = e.target.value;
+    }
+
+    setEditedContentOnChange("separator_color", newColor);
+  };
+
+  return (
+    <>
+      <Separator separator_width={width} separator_color={color} />
+      <h3>Size:</h3>
+      Width:
+      <input
+        className="slider width-range"
+        type="range"
+        min="0"
+        max="100"
+        defaultValue={width + ""}
+        onChange={e =>
```

Comment:
Keep in mind that this may be taxing on the computation side because you would be re-rendering every single tick the user changes the color. First, I would double check that it only calls `onChange` once the user lets go of their mouse. If that is the case, you are good here

If not, I would look into `onMouseUp` or something like that so it only updates once the user has let go.